static/app/components/replays/alerts/replayUnsupportedAlert.tsx

```diff
@@ -8,15 +8,20 @@ interface Props {
 }
 
 export default function ReplayUnsupportedAlert({projectSlug}: Props) {
-  const link = (
-    <ExternalLink href="https://docs.sentry.io/product/session-replay/getting-started/" />
+  const docsLink = (
+    <ExternalLink href="https://docs.sentry.io/product/session-replay/getting-started/#supported-sdks" />
   );
+  const waitlistLink = <ExternalLink href="https://sentry.io/lp/mobile-replay-beta/" />;
   return (
     <Alert icon={<IconInfo />}>
       <strong>{t(`Session Replay isn't available for %s.`, projectSlug)}</strong>{' '}
-      {tct(`To learn more about which SDKs we support, please visit our [link:docs].`, {
-        link,
-      })}
+      {tct(
+        `Currently, [docsLink:Web is supported], and Mobile is being developed. Join our [waitlistLink:waitlist].`,
```

Comment:
```suggestion
        `Mobile is currently being developed, join our [waitlistLink:waitlist]. To learn more about which SDKs we support today, please visit our [docsLink:docs].`,
```

Reply:
We don't know if the selected project is mobile though. It could be native, or something else.
The alternative you gave is also a bit long. Lets see what the current format looks like and consider a change